Mathlib/MeasureTheory/Measure/Doubling.lean

```diff
@@ -27,6 +26,8 @@ This file records basic facts about uniformly locally doubling measures.
 
 @[expose] public section
 
+assert_not_exists Real.rpow_add
```

Comment:
This looks like a much more targeted use of `assert_not_exists` than usual. I'd drop or (maybe generalise) it unless there is a compelling reason to pick on `Real.rpow_add`.